www/js/services.js

```diff
@@ -1,96 +1,243 @@
-angular.module('starter.services', [])
+angular.module('mobility.services', [])
 
-.factory('Chats', function() {
-  // Might use a resource here that returns a JSON array
-
-  // Some fake testing data
-  var chats = [{
+.factory('Incidents', function() {
+  // Traffic API mock data
+  var incidents = [{
     id: 0,
-    name: 'Ben Sparrow',
-    lastText: 'You on your way?',
-    face: 'https://pbs.twimg.com/profile_images/514549811765211136/9SgAuHeY.png'
+    date: 1423529023,
```

Comment:
Note that dates are subject to change. They may not come to you as UNIX timestamps. Keep that in mind as you implement functionality.